src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -348,6 +352,7 @@ private void checkIfGroupsExists(Course course) {
                         return emptyTutorParticipation;
                     });
 
+            // TODO: This could be 1 repository method as the exercise id is provided anyway.
```

Comment:
Just want to support this decision as it is already done in my PR #890 for modeling, text and file upload exercises. The programming exercise can be added there as well.